FF1Lib/asm/1B_B000_StartWithRoutines.asm

```diff
@@ -301,17 +282,39 @@ Hunter:
     STA (btl_ib_charstat_ptr), Y
 Hu_NotClass:
   RTS
-  NOP
-  NOP
-  NOP
-  NOP
-  NOP
-  NOP
-  NOP
-;NOPs added because I couldnt rebuild the entire start with asm file.
-;Had to swap the bugged hunter function, and maintain instruction length to perserve addressing.
-;A0 00 B1 82 AA BD 27 B2 F0 06 A0 0D 11 80 91 80 60 EA EA EA EA EA EA EA     
-  
+
+; Increase damage from talking to NPCs
+FriendshipRune:
+	LDA #<lut_FriendshipRune
+	STA class_lut_a
+	LDA #>lut_FriendshipRune
+	STA class_lut_b
+
+	LDY #$00
+	LDA (btl_ob_charstat_ptr), Y
+	JSR CheckIfClassQuick
+	BNE @done
+		LDA npc_counter
+		STA tmp
+		LDX npc_damage_divider ; back up the divider
+		@divide:
+			LDA npc_damage_divider 
+			BEQ :+
+				LDA tmp
+				LSR
+				STA tmp
+				DEC npc_damage_divider
+				JMP @divide
+		:
+		STX npc_damage_divider ; restore the divider
+		LDY #$09 ; damage
+		LDA (btl_ib_charstat_ptr), Y
+		CLC
+		ADC tmp
```

Comment:
no overflow check?

Reply:
Good point,  I didn't think of that. Would that just be a check of the appropriate flag, then clamp to 255 if it's set?